diff --git a/config/config.go b/config/config.go
index 9bae52d..50ce409 100644
--- a/config/config.go
+++ b/config/config.go
@@ -12,11 +12,13 @@ import (
 )
 
 const (
-	BaseUrl = "https://bigoquiz.com"
-	// When running angular-bigoquiz-client with ng serve: BaseUrl = "http://localhost:4200"
+	// BaseUrl = "https://bigoquiz.com"
+	// When running angular-bigoquiz-client with ng serve:
+	BaseUrl = "http://localhost:4200"
 
-	BaseApiUrl = "https://api.bigoquiz.com"
-	// When running locally: BaseApuUrl = "http://localhost:8080
+	// BaseApiUrl = "https://api.bigoquiz.com"
+	// When running angular-bigoquiz-client with its debug configuration:
+	BaseApiUrl = "http://localhost:8080"
 
 	PART_URL_LOGIN_CALLBACK_GOOGLE   = "callback-google"
 	PART_URL_LOGIN_CALLBACK_GITHUB   = "callback-github"
