From 6acad335daf43ad739afabbe7164dd2b743131bd Mon Sep 17 00:00:00 2001
From: Murray Cumming <murrayc@murrayc.com>
Date: Fri, 3 Nov 2017 22:45:40 +0100
Subject: [PATCH] debugging: Use localhost

---
 bigoquiz.go      | 2 +-
 config/config.go | 5 +++--
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/bigoquiz.go b/bigoquiz.go
index ba4cac6..13c21fb 100644
--- a/bigoquiz.go
+++ b/bigoquiz.go
@@ -33,7 +33,7 @@ func init() {
 	// using the secret key from the configuration file.
 	store = sessions.NewCookieStore([]byte(conf.CookieKey))
 	store.Options.HttpOnly = true
-	store.Options.Secure = true // Only send via HTTPS connections, not HTTP.
+	// store.Options.Secure = true // Only send via HTTPS connections, not HTTP.
 
 	// Gob encoding for gorilla/sessions
 	// Otherwise, we will see errors such as this when calling store.Save():
diff --git a/config/config.go b/config/config.go
index b1dd159..766e67b 100644
--- a/config/config.go
+++ b/config/config.go
@@ -12,8 +12,9 @@ import (
 )
 
 const (
-	BaseUrl = "https://bigoquiz.com"
-	// When running angular-bigoquiz-client with ng serve: BaseUrl = "http://localhost:4200"
+	// BaseUrl = "https://bigoquiz.com"
+	// When running angular-bigoquiz-client with ng serve:
+	BaseUrl = "http://localhost:4200"
 
 	// This file must be downloaded
 	// (via the "DOWNLOAD JSON" link at https://console.developers.google.com/apis/credentials/oauthclient )
-- 
2.14.1

